src/main/java/examples/pingpong/grpc/PingPongGrpcMain.java

```diff
@@ -61,9 +55,12 @@ public static void main(String[] args) {
         // fetch the ledger ID, which is used in subsequent requests sent to the ledger
         String ledgerId = fetchLedgerId(channel);
 
-        // inspect the packages on the ledger and extract the package id of the package containing the PingPong module
-        // this is helpful during development when the package id changes a lot due to likely frequent changes to the DAML code
-        String packageId = detectPingPongPackageId(channel, ledgerId);
+        // fetch the party IDs that got created in the Daml init script
```

Comment:
i would have passed those in as arguments and resolve them outside but don’t feel strongly about this.